exercises/practice/simple-linked-list/.docs/instructions.append.md

```diff
@@ -1,16 +1,87 @@
 # Instructions append
 
+## How this Exercise is Structured in Python
+
+While `stacks` and `queues` can be implemented using `lists`, `collections.deque`, `queue.LifoQueue`, and `multiprocessing.Queue`, this exercise expects a ["Last in, First Out" (`LIFO`) stack][Baeldung: The Stack Data Structure] (_interactive example [here][LIFO Stack]_) using a _custom-made_ [singly linked list][singly linked list].
+
+<br>
+
+_push and pop happen here_
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;|
+
+
+(_head_) **`[Node_3]` -> `[Node_2]` -> `[Node_1]`** (_tail_) -> None
+
+<br>
+
+
+This should not be confused with a [`LIFO` stack using a dynamic array or list][LIFO Stack Array], which may use a `list` underneath.
+Dynamic array based `stacks` have a different `head` position and different time complexity (Big-O) and memory footprint.
+
+<br>
+
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp;  _push and pop happen here_
```

Comment:
… and then the other.

Reply:
This was only included as an example of what the opposite looks like.  I don't want to overly confuse students with this one, since it is not something that will pass the tests for the exercise.  It is only shown to give students a warning of sorts that `stacks` and `queues` can be implemented from either direction.  But same note as above applies here as well.  We can include a diagram, but that has implications for how students who download the exercise will read things.

Reply:
For either this one or the other - please let me know.  It will take about an hour of my time to make custom diagrams and figure out how and where we upload them to include them.  AFIK, we don't support mermaid in our markdown, so doing a mermaid diagram here is out.

Reply:
No great confidence in the ascii art, and the trade off would be the italics that are wanted in the rendering.   But the non-breaking spaces would not be needed as well.

Reply:
I've inserted SVG diagrams.  I've added descriptions, so hopefully that makes them somewhat accessible.  Students who download will either have to follow the links, or open the images to see.

Reply:
I think that is reasonable.

If the location for the image could be `/.exercism/images/linked-list-stack.svg` this would even render in (I think most) local markdown preview solutions for people.  I confirmed for some locally for me.  If it were delivered to the already existing directory of `.exercism` I think that would work pretty well.  It is where we put our meta data anyway.